package.json

```diff
@@ -70,7 +73,10 @@
     "react-debounce-input": "3.2.0",
     "react-virtualized": "9.21.0",
     "react-select": "3.0.3",
-    "styled-components": "^4.1.2"
+    "styled-components": "^4.1.2",
```

Comment:
If needed, add these 3 peer dependencies to README.md: https://github.com/scality/core-ui#how-to-use-it